components/Players/VideoPlayer.vue

```diff
@@ -83,12 +83,7 @@ export default Vue.extend({
       shaka.polyfill.installAll();
       if (shaka.Player.isBrowserSupported()) {
         this.player = new shaka.Player(this.$refs.videoPlayer);
-        // TODO: Remove Shaka's OSD and use our own
```

Comment:
@Camc314 Shouldn't we start using shaka player instead of shaka-player-ui as well?

Reply:
You just need to change the import to have the non-ui version.

Reply:
@mrtimscampi I know, but that change is not in this PR, that's why I mention it :).

Reply:
It's now using 
`import shaka from 'shaka-player/dist/shaka-player.compiled';
`